taxi/timesheet/__init__.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, entries=None, mappings=None, file=None):
         self.file = file
 
     def get_filtered_entries(self, date=None, filter_callback=None,
-                             regroup=False):
+                             regroup=False, exclude_today=False):
```

Comment:
I think it would be better to add a parameter to the filter callback so that you can call it like so: `filter_callback(entry, date)`. Then you could just change the filter callback of `get_entries` and exclude entries from today. That way we keep a single way to filter entries from `get_filtered_entries`.